config.yaml

```diff
@@ -0,0 +1,180 @@
+baseURL: https://notaryproject.dev/
+enableRobotsTxt: true
+
+theme: [docsy]
+
+# Will give values to .Lastmod etc.
+enableGitInfo: true
+
+# Language settings
+contentDir: content/en
+defaultContentLanguage: en
+defaultContentLanguageInSubdir: false
+languageCode: en-us
+# googleAnalytics: UA-00000000-00
+
+# Useful when translating.
+enableMissingTranslationPlaceholders: true
+
+disableKinds: [taxonomy, term]
+
+# Highlighting config
+pygmentsCodeFences: true
+pygmentsUseClasses: false
+# Use the new Chroma Go highlighter in Hugo.
+#pygmentsOptions = "linenos=table"
+# See https://help.farbox.com/pygments.html
+pygmentsStyle: fruity # "tango"
+
+# Image processing configuration.
+imaging:
+  resampleFilter: CatmullRom
+  quality: 75
+  anchor: smart
+
+languages:
+  en:
+    title: Notary Project
+    description: >-
+      A distributed, reliable key-value store for the most critical data of a
+      distributed system
+    languageName: English
+    contentDir: content/en
+    weight: 1
+
+markup:
+  goldmark:
+    renderer:
+      unsafe: true
+  highlight:
+    # See a complete list of available styles at https://xyproto.github.io/splash/docs/all.html
+    style: tango
+    # Uncomment if you want your chosen highlight style used for code blocks without a specified language
+    # guessSyntax: true
+
+# Everything below this are Site Params
+
+params:
+  copyright: Notary Project Authors
+  description: >-
+    A distributed, reliable key-value store for the most critical data of a
+    distributed system
+
+    # Repository configuration (URLs for in-page links to opening issues and suggesting changes)
+  github_repo: https://github.com/notaryproject/notaryproject.dev
+  github_branch: main
+
+  # An optional link to a related project repo. For example, the sibling repository where your product code lives.
+  github_project_repo: https://github.com/notaryproject/notaryproject
+
+  # Enable Algolia DocSearch
+  algolia_docsearch: false
+
+  # Enable Lunr.js offline search
+  offlineSearch: false
+
+  # Enable syntax highlighting and copy buttons on code blocks with Prism
+  prism_syntax_highlighting: false
+
+  # Versions
+
+  # Menu title if your navbar has a versions selector to access old versions of your site.
+  # This menu appears only if you have at least one [params.versions] set.
+  version_menu: Versions
+  versions:
+    latest: v2
+    all:
+      - v2
+      - v1
+    deprecation_warning: |-
+      the documentation is no longer actively maintained.
+        The page that you are viewing is the last archived version.
```

Comment:
I wish that we had cleaned this up first. Maybe we can do the cleanup in this repo and back port a solution.

Reply:
I opened a PR a little bit ago that looks at doing some of that cleanup: https://github.com/etcd-io/website/pull/354